src/main/java/io/fabric8/jenkins/openshiftsync/BuildConfigWatcher.java

```diff
@@ -270,22 +320,23 @@ public Void call() throws Exception {
                         String existingBuildRunPolicy = null;
 
                         BuildConfigProjectProperty buildConfigProjectProperty = job.getProperty(BuildConfigProjectProperty.class);
+
                         if (buildConfigProjectProperty != null) {
-                            existingBuildRunPolicy = buildConfigProjectProperty.getBuildRunPolicy();
-                            long updatedBCResourceVersion = parseResourceVersion(buildConfig);
-                            long oldBCResourceVersion = parseResourceVersion(buildConfigProjectProperty.getResourceVersion());
-                            BuildConfigProjectProperty newProperty = new BuildConfigProjectProperty(buildConfig);
-                            if (updatedBCResourceVersion <= oldBCResourceVersion && newProperty.getUid().equals(buildConfigProjectProperty.getUid()) && newProperty.getNamespace().equals(buildConfigProjectProperty.getNamespace())
-                                    && newProperty.getName().equals(buildConfigProjectProperty.getName()) && newProperty.getBuildRunPolicy().equals(buildConfigProjectProperty.getBuildRunPolicy())) {
-                                return null;
-                            }
-                            buildConfigProjectProperty.setUid(newProperty.getUid());
-                            buildConfigProjectProperty.setNamespace(newProperty.getNamespace());
-                            buildConfigProjectProperty.setName(newProperty.getName());
-                            buildConfigProjectProperty.setResourceVersion(newProperty.getResourceVersion());
-                            buildConfigProjectProperty.setBuildRunPolicy(newProperty.getBuildRunPolicy());
+                          existingBuildRunPolicy = buildConfigProjectProperty.getBuildRunPolicy();
```

Comment:
per the earlier exchange, yeah. I'm claiming precedence here .... fix your tabbing so that it is 4 spaces

Reply:
still waiting on this @waveywaves